markdown/developers/tutorials/unofficial_pipelines.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Tutorial: Reusing nf-core components in non-nf-core pipelines. "
+subtitle: Guidance on how to use nf-core code and best practices in non-nf-core pipelines.
+---
+
+While we highly recommend and promote contributing to existing nf-core pipelines, in some cases you may disagree with some of the design decisions, or only wish to follow _parts_ of nf-core best practices for your own non-nf-core pipeline. This tutorial describes how to use nf-core code without making official nf-core pipelines.
```

Comment:
```suggestion
While we highly recommend and promote contributing to existing nf-core pipelines, in some cases this may not be possible or appropriate. In this case we hope that it will be helpful to use _parts_ of nf-core best practices for your own non-nf-core pipeline. This tutorial describes how to use nf-core code without making official nf-core pipelines.
```